wallet-server-ui/src/type/wallet-server-types.ts

```diff
@@ -119,6 +120,11 @@ export const enum WalletMessageType {
 
   importseed = 'importseed',
   importxprv = 'importxprv',
+  listwallets = 'listwallets',
```

Comment:
I think this needs to be removed, i'm getting this error while trying to test this morning

![Screenshot from 2022-07-01 07-31-06](https://user-images.githubusercontent.com/3514957/176894891-384fb1e0-470c-44b7-8fe3-a109eff4bb58.png)